docs/Advanced-Topics-Custom-Block-Render.md

```diff
@@ -0,0 +1,152 @@
+---
+id: advanced-topics-custom-block-render-map
+title: Custom Block Rendering
+layout: docs
+category: Advanced Topics
+next: advanced-topics-block-components
+permalink: docs/advanced-topics-custom-block-render-map.html
+---
+
+This article discusses how to customize Draft default block rendering.
+The block rendering is used to define supported block types and their respective
+renderers, as well as converting pasted content to known Draft block types.
+
+When pasting content or when using the
+[convertFromHTML](https://facebook.github.io/draft-js/docs/api-reference-data-conversion.html#convertfromhtml)
+Draft will then convert the pasted content to the respective block rendering type
+by matching the Draft block render map with the matched tag.
+
+## Draft default block render map
+
+|  HTML element   |            Draft block type             |
+| --------------- | --------------------------------------- |
+|     `<h1/>`     |               header-one                |
+|     `<h2/>`     |               header-two                |
+|     `<h3/>`     |              header-three               |
+|     `<h4/>`     |               header-four               |
+|     `<h5/>`     |               header-five               |
+|     `<h6/>`     |               header-six                |
+|     `<h6/>`     |               header-six                |
+| `<blockquote/>` |               blockquote                |
+|    `<pre/>`     |               code-block                |
+|   `<figure/>`   |                 atomic                  |
+|     `<li/>`     | unordered-list-item,ordered-list-item** |
+|    `<div/>`     |               unstyled***               |
+
```

Comment:
For some reason this table is still not aligned in the rendered documentation. Also we can omit the dash after the asterisks in the comments below the table.
<img width="550" alt="screen shot 2016-08-27 at 6 52 44 pm" src="https://cloud.githubusercontent.com/assets/1114467/18031227/1baafe1a-6c8b-11e6-84fe-332ea4d043b3.png">